luthier/src/test/resources/snapshots/.snapshotsHistory.json

```diff
@@ -1 +1 @@
-["2019_07_Jul_22_Mon_11:43:02/","2019_07_Jul_24_Wed_17:19:43/","2019_07_Jul_25_Thu_11:43:48/"]
\ No newline at end of file
+["2019_07_Jul_30_Tue_11:41:36/","2019_07_Jul_30_Tue_11:52:20/","2019_07_Jul_30_Tue_16:31:45/","2019_07_Jul_30_Tue_16:41:22/","2019_07_Jul_30_Tue_16:42:21/","2019_07_Jul_30_Tue_20:24:10/","2019_07_Jul_31_Wed_14:32:32/","2019_07_Jul_31_Wed_15:38:54/"]
```

Comment:
None of this should be in source control. We should add `src/test/resources/snapshots` to the .gitignore file.

Reply:
Oh sure. 
This does clog the commit reviewable files